### PR TITLE
Rov 72 generate an angular component named registration 2

### PR DESCRIPTION
**Pull request by Vedant-Tyagi**
Made the angular  component .

### CODE REVIEW

File: src/app/application/registration 2/registration 2.component.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-registration',
+    templateUrl: './registration 2.component.html',
+    styleUrls: ['./registration 2.component.css']
+  })
+
+export class registration {
```
Comment (yaneshtyagi) [inline src/app/application/registration 2/registration 2.component.ts:9]: 1. Never ever have space in source code file names or folders. Rename it to `registration` or `registration-v2` if you must have version numbers.
2. Registration must go inside the `signup` module.

**Review by yaneshtyagi — CHANGES_REQUESTED** [review]

File: src/app/application/registration 2/registration 2.component.ts (hunk 1 above)
Comment (rachittyagi) [reply src/app/application/registration 2/registration 2.component.ts:9]: @yaneshtyagi signup module is different, this registration-v2 is for learning purpose. We will leavrage this later user profile component.

Comment (yaneshtyagi) [reply src/app/application/registration 2/registration 2.component.ts:9]: OK.

**Review by rachittyagi — APPROVED** [review]

File: src/app/application/registration 2/registration 2.component.ts (hunk 1 above)
Comment (Vedant-Tyagi) [reply src/app/application/registration 2/registration 2.component.ts]: Changed the name of the component

**Review by rachittyagi — APPROVED** [review]

**Pull request merged**